benchmarks/transformer/score_mod.py

```diff
@@ -176,6 +186,46 @@ def calculate_speedup(results: ExperimentResults, type: str) -> float:
         raise ValueError(f"Invalid type {type}")
 
 
+def calculate_bandwidth(
+    config: ExperimentConfig, results: ExperimentResults, type: str
+) -> float:
+    if type == "fwd":
+        batch_size, q_heads, q_seq_len, kv_heads, kv_seq_len, head_dim = config.shape
+        query_size = (
+            batch_size
+            * q_heads
+            * q_seq_len
+            * head_dim
+            * torch.finfo(config.dtype).bits
+            / 8
+        )
+        kv_size = (
+            batch_size
+            * kv_heads
+            * kv_seq_len
+            * head_dim
+            * torch.finfo(config.dtype).bits
+            / 8
+            * 2
+        )
+        output_size = query_size
+        total_size = (query_size + kv_size + output_size) / 1024 / 1024 / 1024  # In GB
+        time_in_seconds = results.fwd_times.compiled_time / 1e6
+        return total_size / time_in_seconds / 1024
+    else:
+        raise ValueError(f"Invalid type {type}")
+
+
+def calculate_gflops(config: ExperimentConfig, results: ExperimentResults) -> float:
```

Comment:
same tflops comment but not a big deal and a nit

Reply:
I messed up ghstack, therefore it's fixed in next PR in stack: https://github.com/pytorch/pytorch/pull/130854. I'll land that one directly.

Reply:
Okay I think I messed this up even more... I'll merge this https://github.com/pytorch/pytorch/pull/130850 instead.